hooks/010-locale-vconsole-symlinks.chroot

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -eu
+
+# setup default console configuration, make them symlinks
+# into writable so they can be changed
+printf 'LANG=C.UTF-8\n' > /etc/writable/locale.conf
+ln -s ../writable/locale.conf /etc/default/locale
+ln -s writable/locale.conf /etc/locale.conf
+
+# setup default vconsole/keyboard configuration, make it a
+# symlink as well so it can change
+mv /etc/vconsole.conf /etc/writable/vconsole.conf
+ln -s ../writable/vconsole.conf /etc/default/keyboard
+ln -s writable/vconsole.conf /etc/vconsole.conf
```

Comment:
What is vconsole.conf here? In current core24 revisions it is a broken symlink to `default/keyboard`, will this fix that?